src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -0,0 +1,185 @@
+      <remarks>
+        This transform produces a matrix of token ngrams/skip-grams counts for a given corpus of text.
+        The n-grams are represented as count vectors, with vector slots corresponding to n-grams.
+        Embedding ngrams in a vector space allows their contents to be compared in an efficient manner. 
+        The slot values in the vector can be weighted by the following factors:
+        <list>
+          <item>
+            <description>term frequency - The number of occurrences of the slot in the text</description>
+          </item>
+          <item>
+            <description>
+              inverse document frequency - A ratio (the logarithm of inverse relative slot frequency)
+              that measures the information a slot provides by determining how common or rare it is across the entire text.
+            </description>
+          </item>
+            <item>
+              <description>term frequency-inverse document frequency - the product term frequency and the inverse document frequency.</description>
+            </item>
+        </list>
+        This transform is not typically used on its own, but it is one of the transforms composing the <see cref="Microsoft.ML.Transforms.TextFeaturizer">Text Featurizer</see> .
+      </remarks>
+      <seealso cref="Microsoft.ML.Transforms.WordTokenizer"/>
+      <seealso cref="Microsoft.ML.Transforms.TextToKey"/>
+      <seealso cref="Microsoft.ML.Transforms.TextFeaturizer"/>
+      <seealso cref="Microsoft.ML.Transforms.CharacterTokenizer"/>
+      <example>
+        <code>
+          pipeline.Add(new NGramTranslator(&quot;TextColumn&quot;){ Weighting=NgramTransformWeightingCriteria.TfIdf  } );
+      </code>
+      </example>
+    </member>
+
+    <member name="SentimentAnalyzer">
+      <summary>
+        Uses a pretrained sentiment model to score input strings.
+      </summary>
+      <remarks>
+        <para>The Sentiment transform returns the probability that the sentiment of a natural text is positive. </para>
+        <para>
+          The model was trained with the <a href="http://anthology.aclweb.org/P/P14/P14-1146.pdf">Sentiment-specific word embedding (SSWE)</a>  and NGramFeaturizer on Twitter sentiment data,
+          similarly to the sentiment analysis part of the
+          <a href="https://www.microsoft.com/cognitive-services/en-us/text-analytics-api">Text Analytics cognitive service</a>. 
+          The transform outputs a score between 0 and 1 as a sentiment prediction 
+          (where 0 is a negative sentiment and 1 is a positive sentiment).</para> 
+          <para>Currently it supports only English.</para>
+      </remarks>
+      <example>
+        <code>
+          pipeline.Add(new SentimentAnalyzer(){ Source = &quot;TextColumn&quot; }  );
+        </code>
+      </example>
+    </member>
+
+    <member name="CharacterTokenizer">
+      <summary>
+        Character-oriented tokenizer where text is considered a sequence of characters. 
```

Comment:
>Character-oriented [](start = 8, length = 18)

There's something that seems a bit off about this description. "Character-oriented" is a strange description, in any case text is as far as I know always considered a sequence of characters, and at the end of this I don't really have a clear idea of what it's doing.

Maybe, "breaks text into individual tokens consisting of each individual character" might be a good description?